pkg/kwokctl/runtime/kind/cluster.go

```diff
@@ -378,11 +378,50 @@ func (c *Cluster) addKind(ctx context.Context, env *env) (err error) {
 		return err
 	}
 
+	componentPatches := make([]internalversion.ComponentPatches, 0, len(conf.ExtraArgs))
+	componentNames := make(map[string]int)
+	for i, extraArgs := range conf.ExtraArgs {
+		splitedArgs := strings.SplitN(extraArgs, "=", 3)
+		if len(splitedArgs) != 3 {
+			return fmt.Errorf("invalid extraArgs: %s. It should be in the format of component=arg=value", extraArgs)
+		}
+		if n, ok := componentNames[splitedArgs[0]]; ok {
+			componentPatches[n].ExtraArgs = append(componentPatches[n].ExtraArgs, internalversion.ExtraArgs{
+				Key:   splitedArgs[1],
+				Value: splitedArgs[2],
+			})
+			continue
+		}
+		componentPatches = append(componentPatches, internalversion.ComponentPatches{
+			Name: splitedArgs[0],
+			ExtraArgs: []internalversion.ExtraArgs{
+				{
+					Key:   splitedArgs[1],
+					Value: splitedArgs[2],
+				},
+			},
+		})
+		componentNames[splitedArgs[0]] = i
+	}
```

Comment:
I think we can move this part into main and add a convert from string to ComponentPatches, like
https://github.com/kubernetes-sigs/kwok/blob/b8917eccea1e0eb80af5074787573d86a6ca3231/pkg/kwokctl/cmd/create/cluster/cluster.go#L190

Reply:
I've done.